mkdocs/docs/configuration.md

```diff
@@ -372,7 +372,73 @@ Specific headers defined by the RESTCatalog spec include:
 
 | Key                                  | Options                               | Default              | Description                                                                                                                                                                                        |
 | ------------------------------------ | ------------------------------------- | -------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `header.X-Iceberg-Access-Delegation` | `{vended-credentials,remote-signing}` | `vended-credentials` | Signal to the server that the client supports delegated access via a comma-separated list of access mechanisms. The server may choose to supply access via any or none of the requested mechanisms |
+| `header.X-Iceberg-Access-Delegation` | `{vended-credentials,remote-signing}` | `vended-credentials` | Signal to the server that the client supports delegated access via a comma-separated list of access mechanisms. The server may choose to supply access via any or none of the requested mechanisms. When using `vended-credentials`, the server provides temporary credentials to the client. When using `remote-signing`, the server signs requests on behalf of the client. |
+
+#### Authentication Options
+- **SigV4**: For AWS services that require SigV4 signing.
+- **Token**: Use the `token` property to pass a bearer token for services that accept token-based authentication.
+- **Credential**: Use the `credential` property with format `client_id:client_secret` for authentication.
+- **OAuth2**: Use the `oauth2-server-uri` property to specify a custom OAuth2 endpoint for client credentials authentication.
+
+#### Common Integrations & Examples
+
+##### Glue (AWS)
+```yaml
+catalog:
+  s3_tables_catalog:
+    type: rest
+    uri: https://glue.<region>.amazonaws.com/iceberg
+    warehouse: <account-id>:s3tablescatalog/<table-bucket-name>
+    rest.sigv4-enabled: true
+    rest.signing-name: glue
+    rest.signing-region: <region>
+```
+
+##### Unity Catalog (Databricks)
```

Comment:
i think this should work for both databricks UC and oss UC https://github.com/unitycatalog/unitycatalog
```suggestion
##### Unity Catalog
```